Neos.Flow/Tests/Unit/Validation/ValidatorResolverTest.php

```diff
@@ -474,19 +489,27 @@ public function buildBaseValidatorConjunctionAddsValidatorsOnlyForPropertiesHold
 
         $mockObjectManager = $this->getMockBuilder(ObjectManagerInterface::class)->disableOriginalConstructor()->getMock();
         $mockObjectManager->expects(self::any())->method('isRegistered')->will(self::returnValue(true));
-        $mockObjectManager->expects(self::at(1))->method('getScope')->with($entityClassName)->will(self::returnValue(Configuration::SCOPE_PROTOTYPE));
-        $mockObjectManager->expects(self::at(3))->method('getScope')->with($otherClassName)->will(self::returnValue(null));
+        $mockObjectManager->expects(self::exactly(2))->method('getScope')->withConsecutive([$entityClassName], [$otherClassName])->willReturnOnConsecutiveCalls(Configuration::SCOPE_PROTOTYPE, null);
 
         $mockReflectionService = $this->createMock(ReflectionService::class);
         $mockReflectionService->expects(self::any())->method('getAllImplementationClassNamesForInterface')->with(PolyTypeObjectValidatorInterface::class)->will(self::returnValue([]));
         $mockReflectionService->expects(self::any())->method('getClassPropertyNames')->will(self::returnValue(['entityProperty', 'otherProperty']));
-        $mockReflectionService->expects(self::at(1))->method('getClassSchema')->will(self::returnValue(null));
-        $mockReflectionService->expects(self::at(2))->method('getPropertyTagsValues')->with($modelClassName, 'entityProperty')->will(self::returnValue(['var' => [$entityClassName]]));
-        $mockReflectionService->expects(self::at(3))->method('isPropertyAnnotatedWith')->will(self::returnValue(false));
-        $mockReflectionService->expects(self::at(4))->method('getPropertyAnnotations')->with($modelClassName, 'entityProperty', Annotations\Validate::class)->will(self::returnValue([]));
-        $mockReflectionService->expects(self::at(5))->method('getPropertyTagsValues')->with($modelClassName, 'otherProperty')->will(self::returnValue(['var' => [$otherClassName]]));
-        $mockReflectionService->expects(self::at(6))->method('isPropertyAnnotatedWith')->will(self::returnValue(false));
-        $mockReflectionService->expects(self::at(7))->method('getPropertyAnnotations')->with($modelClassName, 'otherProperty', Annotations\Validate::class)->will(self::returnValue([]));
+        $mockReflectionService->expects(self::any())->method('getClassSchema')->will(self::returnValue(null));
```

Comment:
```suggestion
```
any can be skipped and mocked method returns null by default.

Reply:
True.